pyanaconda/payload/__init__.py

```diff
@@ -1167,7 +1167,8 @@ def _setupMedia(self, device):
 
         # Check to see if the device is already mounted, in which case
         # we don't need to mount it again so skip this part
-        elif not (method.method == "cdrom" and INSTALL_TREE in blivet.util.get_mount_paths(device.path)):
+        elif not (method.method == "cdrom" and (INSTALL_TREE in blivet.util.get_mount_paths(device.path)
+                                                or DRACUT_REPODIR in blivet.util.get_mount_paths(device.path))):
```

Comment:
You are over 99 characters limit and this elif is not really readable. Could you please rewrite this to something more sane? For example to abstract this to a function or split to multiple if's.

Reply:
Done